app/src/main/java/com/nervousfish/nervousfish/modules/util/AUtils.java

```diff
@@ -0,0 +1,23 @@
+package com.nervousfish.nervousfish.modules.util;
+
+/**
+ * This class should be extended by all platform-specific {@link IUtils} implementations and contains
+ * all cross-platform functionality
+ */
+@SuppressWarnings("PMD.AbstractClassWithoutAnyMethod")
+abstract class AUtils implements IUtils {
```

Comment:
Why does this Abstract Utils class only do something with QR, it isn't really a "utility" class, right?